apps/ern-ithaca/src/components/AppFooter.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <PageFooter>
+    <div class="footer-container width-full">
+      <div class="footer-content width-medium">
+        <div class="footer-column footer-links">
+          <p><strong>ERN Ithaca</strong></p>
+          <UnorderedList listType="none">
+            <li><router-link :to="{ name: 'home' }">Home</router-link></li>
+            <li><router-link :to="{ name: 'about' }">About</router-link></li>
+            <li>
+              <router-link :to="{ name: 'dashboard' }">Dashboard</router-link>
+            </li>
+            <li>
+              <router-link :to="{ name: 'documents' }">Documents</router-link>
+            </li>
+            <li>
+              <router-link :to="{ name: 'transparency' }"
+                >Transparency Statement</router-link
+              >
+            </li>
+            <!-- more links here -->
+          </UnorderedList>
+        </div>
+        <div class="footer-column footer-links">
+          <p><strong>For Members</strong></p>
+          <UnorderedList listType="none">
+            <li>
+              <a href="/apps/central/#/admin/users">Sign in</a>
```

Comment:
Why is this linking to the admin user page? I'd expect this link to open the sign in modal, just like the sign in button at the top of the page.

Reply:
I'm removing this. I tried to implement this earlier, but it was decided that it wasn't necessary to have a sign in link in the footer.